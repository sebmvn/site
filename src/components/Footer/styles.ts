import styled, { css } from 'styled-components'

import media from 'styled-media-query'

export const Wrapper = styled.footer`
  ${({ theme }) => css`
    background: ${theme.colors.footerBg};
    padding-top: ${theme.spacings.large};

    ${media.lessThan('medium')`
      padding-top: ${theme.spacings.medium};
    `}
  `}
`

export const Content = styled.div`
  display: flex;
  flex-direction: column;
  align-items: center;
`

export const Column = styled.div`
  ${({ theme }) => css`
    a,
    span {
      display: block;
      text-decoration: none;
      font-size: ${theme.font.sizes.medium};
    }

    a {
      word-wrap: break-word;
      overflow-wrap: break-word;
    }

    a:hover {
      text-decoration: underline;
    }

    ${media.lessThan('medium')`
      display: flex;
      flex-direction: column;
      align-items: center;
      text-align: center;

     p {
        font-size: ${theme.font.sizes.xsmall};
      }
    `}
  `}
`
export const Info = styled.div`
  ${({ theme }) => css`
    color: ${theme.colors.text};
    font-size: ${theme.font.sizes.small};
    margin-bottom: ${theme.spacings.xxlarge};

    > :last-child {
      margin-top: ${theme.spacings.xsmall};
    }

    ${media.lessThan('medium')`

    & .mapouter{
      iframe{

        max-width: 100%;
      }
    }
    `}
  `}
`

export const SocialMedia = styled.nav`
  ${({ theme }) => css`
    display: flex;
    align-items: center;

    margin-bottom: ${theme.spacings.small};

    a {
      padding: ${theme.spacings.xxsmall};
      margin-right: ${theme.spacings.xsmall};

      background: ${theme.colors.text};
      border-radius: 100%;

      svg {
        width: 2.4rem;
        stroke-width: 0.2rem;
        color: ${theme.colors.footerBg};
      }

      ${media.lessThan('medium')`
        margin-right: ${theme.spacings.xxsmall};
      `}
    }
  `}
`

export const ContactButton = styled.a`
  text-decoration: none;

  :hover {
    text-decoration: none;
  }
`

export const Menu = styled.nav`
  ${({ theme }) => css`
    a {
      margin-bottom: ${theme.spacings.xxsmall};

      font-size: ${theme.font.sizes.small};
      font-weight: ${theme.font.bold};

      color: ${theme.colors.assets100};
    }
  `}
`

export const Image = styled.img`
  ${({ theme }) => css`
    width: 18rem;
    margin-top: ${theme.spacings.xxsmall};
  `}
`

export const Copyright = styled.div`
  ${({ theme }) => css`
    color: ${theme.colors.text};
    background-color: ${theme.colors.copyrightBg};
    font-size: ${theme.font.sizes.xsmall};

    padding: ${theme.spacings.xsmall} 0;

    margin-top: ${theme.spacings.large};
    text-align: center;
  `}
`
