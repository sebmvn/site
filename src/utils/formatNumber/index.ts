export const formatPriceNotation = (value: string) => {
  value = value?.replace('.', '').replace(',', '').replace(/\D/g, '')

  if (isNaN(parseFloat(value))) {
    return ''
  }

  const options = { minimumFractionDigits: 2 }
  const result = new Intl.NumberFormat('pt-BR', options).format(
    parseFloat(value) / 100
  )

  return result
}

export const formatIntegerNotation = (value: string) => {
  return value?.replace('.', '').replace(',', '').replace(/\D/g, '')
}

export const toDecimal = (value: string) => {
  const formattedValue = formatIntegerNotation(value)
  return parseFloat(formattedValue!) / 100
}

export const toInteger = (value: string) => {
  const formattedValue = formatIntegerNotation(value)
  return parseInt(formattedValue!)
}
