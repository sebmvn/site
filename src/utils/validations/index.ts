import { CalendarType } from 'hooks/use-calc'
import Joi from 'joi'

const fieldsValidations = {
  initialValue: Joi.string().required().messages({
    'any.required': 'Campo obrigatório',
    'string.empty': 'Campo obrigatório',
    'string.min': 'tamanho minimo de 1 digito',
    'string.base': 'o tipo do dado precisa ser textual'
  }),
  monthlyValue: Joi.string().required().messages({
    'any.required': 'Campo obrigatório',
    'string.empty': 'Campo obrigatório',
    'string.min': 'tamanho minimo de 1 digito',
    'string.base': 'o tipo do dado precisa ser textual'
  }),

  interestRate: Joi.string().required().messages({
    'any.required': 'Campo obrigatório',
    'string.empty': 'Campo obrigatório',
    'string.min': 'tamanho minimo de 1 digito',
    'string.base': 'o tipo do dado precisa ser textual'
  }),
  period: Joi.string().required().messages({
    'any.required': 'Campo obrigatório',
    'string.empty': 'Campo obrigatório',
    'string.min': 'tamanho minimo de 1 digito',
    'string.base': 'o tipo do dado precisa ser textual'
  }),
  interestRateCalendar: Joi.string().valid('monthly', 'yearly').required(),
  periodCalendar: Joi.string().valid('monthly', 'yearly').required()
}

export type FieldErrors = {
  [key: string]: string
}

const getFieldErrors = (objError: Joi.ValidationResult) => {
  const errors: FieldErrors = {}

  if (objError.error) {
    objError.error.details.forEach((err) => {
      errors[err.path.join('.')] = err.message
    })
  }

  return errors
}

type compoundInterestCalculatorFields = {
  initialValue: string
  monthlyValue: string
  interestRate: string
  period: string
  interestRateCalendar: CalendarType
  periodCalendar: CalendarType
}
export const compoundInterestCalculatorValidate = (
  values: compoundInterestCalculatorFields
) => {
  const schema = Joi.object(fieldsValidations)

  return getFieldErrors(schema.validate(values, { abortEarly: false }))
}
