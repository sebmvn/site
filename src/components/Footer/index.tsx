import Logo from 'components/Logo'
import Button from 'components/Button'

import * as S from './styles'

import { useTheme } from 'styled-components'

import { Instagram as InstagramIcon } from '@styled-icons/feather/Instagram'
import { Facebook as FacebookIcon } from '@styled-icons/feather/Facebook'
import { Youtube as YoutubeIcon } from '@styled-icons/feather/Youtube'
import { Container } from 'components/Container'

const currentYear = new Date().getFullYear()

const Footer = () => {
  const theme = useTheme()

  return (
    <S.Wrapper id="contato">
      <Container>
        <Logo color="primary" size="large" />
        <S.Content>
          <S.Info>
            <div className="mapouter">
              <div className="gmap_canvas">
                <iframe
                  width="1080"
                  height="540"
                  id="gmap_canvas"
                  src="https://maps.google.com/maps?q=vice%20almirante%20abreu,568%20&t=&z=16&ie=UTF8&iwloc=&output=embed"
                  frameBorder="0"
                  scrolling="no"
                ></iframe>
              </div>
            </div>

            <p>
              Rua Vice Almirante Abreu, 568 - Centro, Rio Grande-RS CEP
              96200-270
            </p>
          </S.Info>
          <S.SocialMedia>
            <a
              href="https://www.instagram.com/sebmvnoficial/"
              target="_blank"
              rel="noopenner, noreferrer"
            >
              <InstagramIcon />
            </a>
            <a
              href="https://www.facebook.com/sebmvn.org"
              target="_blank"
              rel="noopenner, noreferrer"
            >
              <FacebookIcon fill={theme.colors.footerBg} />
            </a>
            <a
              href="https://www.youtube.com/c/SEBMVN"
              target="_blank"
              rel="noopenner, noreferrer"
            >
              <YoutubeIcon />
            </a>
          </S.SocialMedia>

          <S.ContactButton
            href="https://api.whatsapp.com/send?phone=5553981118888"
            target="_blank"
            rel="noreferrer"
          >
            <Button size="medium">Fale conosco</Button>
          </S.ContactButton>

          {/* <S.Column aria-labelledby="resources" data-testid="resources">
            <MenuNav>
              <MenuLink href="/#sobre">Sobre Nós</MenuLink>
              <MenuLink href="/#atividades">Atividades</MenuLink>
              <MenuLink href="/#psicografias">Psicografias</MenuLink>
              <MenuLink href="/#obras-basicas">Obras Básicas</MenuLink>
              <MenuLink href="/#contato">Contato</MenuLink>
            </MenuNav>
          </S.Column> */}
          {/* <ArrowUpCircle onClick={() => scroll('inicio')} width={30} /> */}
        </S.Content>
      </Container>

      <S.Copyright>
        <Container>
          Copyright © {currentYear} | Desenvolvido com{' '}
          <svg
            width="10"
            height="10"
            viewBox="0 0 22 18"
            fill="none"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M11.0291 4.80001C10.5737 3.26737 8.6575 2.44211 7.75629 2.14737C7.75629 2.14737 5.63532 1.81228 4.48349 2.58947C3.3564 3.34998 2.49136 4.3698 2.20676 5.38948C1.85052 6.6659 1.96048 7.71672 2.49136 8.9263C3.20282 10.5474 6.47563 12.9053 6.47563 12.9053C6.47563 12.9053 9.362 14.849 11.0291 16C13.4742 14.2159 14.8422 13.5522 17.1478 11.5789L17.2099 11.5258C18.1473 10.7238 18.7938 10.1707 19.4245 9.07367C20.0937 7.9098 20.0954 6.42106 19.8514 5.53685C19.5668 4.50527 18.9633 3.63122 18.0016 2.88421C16.8632 2 15.3928 2.09825 14.4442 2C12.0536 2.70737 11.2188 4.11229 11.0291 4.80001Z"
              stroke="#0D2538"
              strokeOpacity="0.8"
              strokeWidth="3"
              strokeLinejoin="round"
            />
          </svg>{' '}
          pelo setor de mídia da SEBMVN.
        </Container>
      </S.Copyright>
    </S.Wrapper>
  )
}
export default Footer
