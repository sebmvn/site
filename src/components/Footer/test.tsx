import { screen } from '@testing-library/react'
import { renderWithTheme } from 'utils/test-utils'
import Footer from '.'

describe('<Footer />', () => {
  it('should render 2 column topics', () => {
    const { container } = renderWithTheme(<Footer />)

    expect(screen.getByTestId(/contact/i)).toBeInTheDocument()
    expect(screen.getByTestId(/resources/i)).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
