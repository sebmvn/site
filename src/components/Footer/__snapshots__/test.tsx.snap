// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render 2 column topics 1`] = `
.c2 {
  color: #193241;
  width: 20rem;
  height: 5.9rem;
}

.c8 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  border: 0;
  cursor: pointer;
  text-transform: uppercase;
  border-radius: 0.5rem;
  padding: 0.8rem;
  -webkit-text-decoration: none;
  text-decoration: none;
  color: #193241;
  font-weight: 900;
  font-family: Poppins,-apple-system,BlinkMacSystemFont,'Segoe UI',Roboto,Oxygen,Ubuntu,Cantarell,'Open Sans','Helvetica Neue',sans-serif;
  height: 4rem;
  font-size: 1.4rem;
  padding: 0.8rem 3.2rem;
  background: #0ADC7D;
  color: #193241;
}

.c8:hover {
  background: linear-gradient( 180deg,#0ADC7D 0%,#00823c 100% );
}

.c0 {
  background: #F4F4F4;
  padding-top: 4.0rem;
}

.c3 {
  display: grid;
  grid-template-columns: 2fr 1fr;
  gap: 3.2rem;
  margin-top: 2.4rem;
}

.c4 a,
.c4 span {
  display: block;
  -webkit-text-decoration: none;
  text-decoration: none;
  font-size: 1.6rem;
}

.c4 a {
  word-wrap: break-word;
  overflow-wrap: break-word;
}

.c4 a:hover {
  -webkit-text-decoration: underline;
  text-decoration: underline;
}

.c5 {
  color: #193241;
  font-size: 1.4rem;
  margin-bottom: 5.6rem;
}

.c5 > :last-child {
  margin-top: 1.6rem;
}

.c6 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  margin-bottom: 2.4rem;
}

.c6 a {
  padding: 0.8rem;
  margin-right: 1.6rem;
  background: #193241;
  border-radius: 100%;
}

.c6 a svg {
  width: 2.4rem;
  stroke-width: 0.2rem;
  color: #F4F4F4;
}

.c9 a {
  margin-bottom: 0.8rem;
  font-size: 1.4rem;
  font-weight: 600;
  color: #193241;
}

.c10 {
  width: 18rem;
  margin-top: 0.8rem;
}

.c11 {
  color: #193241;
  background-color: #ECECEC;
  font-size: 1.2rem;
  padding: 1.6rem 0;
  margin-top: 4.0rem;
  text-align: center;
}

.c7 {
  display: inline-block;
  vertical-align: middle;
  overflow: hidden;
}

.c1 {
  width: 100%;
  max-width: 130rem;
  margin-left: auto;
  margin-right: auto;
  padding-left: calc(3.2rem / 2);
  padding-right: calc(3.2rem / 2);
}

@media (max-width:768px) {
  .c3 {
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
    -webkit-flex-direction: column;
    -ms-flex-direction: column;
    flex-direction: column;
  }
}

@media (max-width:768px) {
  .c4 {
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
    -webkit-flex-direction: column;
    -ms-flex-direction: column;
    flex-direction: column;
    -webkit-align-items: center;
    -webkit-box-align: center;
    -ms-flex-align: center;
    align-items: center;
    text-align: center;
  }

  .c4 p {
    font-size: 1.2rem;
  }
}

@media (max-width:768px) {
  .c6 a {
    margin-right: 0.8rem;
  }
}

<footer
  class="c0"
>
  <div
    class="c1"
  >
    <div
      class="c2"
      color="primary"
    >
      <svg
        aria-label="Hub do Investidor"
        fill="none"
        role="img"
        viewBox="0 0 152 40"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M36.297 29.092V2.741c0-2.674-3.444-3.76-4.98-1.57L11.758 29.092H36.3h-.002Z"
          fill="#0ADC7D"
        />
        <path
          d="M0 29.091v8.198c0 2.646 3.407 3.72 4.926 1.552l6.83-9.75H0Z"
          fill="#6E82FA"
        />
        <path
          class="text"
          d="M84.875 26.715c0 1.074-.707 1.549-1.904 1.549-1.198 0-1.904-.475-1.904-1.549v-2.532c0-1.058.706-1.536 1.904-1.536 1.197 0 1.904.475 1.904 1.536v2.532ZM82.97 21.65c-1.55 0-3.04.537-3.04 2.517v2.579c0 1.98 1.487 2.517 3.04 2.517 1.552 0 3.04-.537 3.04-2.517v-2.579c0-1.98-1.488-2.517-3.04-2.517ZM57.68 27.135h-1.467a.804.804 0 0 1-.805-.804v-4.505h-2.025v5.108c0 1.351.769 2.333 2.426 2.333.123 0 .293-.016.43-.062l1.444-1.197v1.09h2.027v-7.274h-2.028v5.308l-.002.003ZM65.8 26.578c0 .706-.352 1.058-1.136 1.058-.784 0-1.136-.352-1.136-1.058v-2.792h1.467c.445 0 .804.36.804.804v1.988ZM65.4 21.65c-.123 0-.29.016-.429.062l-1.441 1.197v-4.726h-2.028v8.591c0 1.98 1.611 2.486 3.163 2.486 1.552 0 3.164-.506 3.164-2.486v-2.792c0-1.35-.769-2.332-2.426-2.332M49.553 22.727h-3.965v-4.54h-2.025v10.907h2.025v-4.54h3.965v4.54h2.028V18.186h-2.028v4.541ZM120.365 25.059l-1.472-.2c-.769-.106-.969-.476-.969-.89v-.305c0-.722.522-.966 1.765-.966.846 0 1.966.14 1.966.14V21.81c-.583-.062-1.244-.154-2.133-.154-1.734 0-2.657.644-2.657 1.95v.49c0 .95.691 1.625 1.704 1.764l1.475.198c.768.108.966.475.966.891v.321c0 .707-.521.95-1.873.95-.828 0-2.241-.169-2.241-.169v1.043s1.12.17 2.441.17c1.827 0 2.732-.645 2.732-1.948v-.49c0-.953-.676-1.626-1.704-1.765M114.045 24.934h-3.593v-.858c0-1.027.722-1.425 1.812-1.425 1.259 0 1.781.552 1.781 1.395v.888Zm-1.781-3.282c-2.287 0-2.948 1.058-2.948 2.563v2.286c0 2.165 1.612 2.76 3.272 2.76.843 0 1.549-.06 2.348-.169v-1.058c-.645.077-1.595.154-2.24.154-1.136 0-2.241-.277-2.241-1.548v-.676h4.728v-1.749c0-1.505-.629-2.563-2.917-2.563M104.909 28.07h-.155l-2.055-6.244h-1.182l2.503 7.271h1.626l2.503-7.27h-1.182l-2.058 6.243ZM97.787 21.652c-.123 0-.29.015-.429.061l-1.889 1.32v-1.212h-1.135v7.274h1.135v-5.186l1.689-.934c.23-.124.521-.185.722-.185.83 0 1.197.413 1.197 1.412v4.895h1.136V23.99c0-1.35-.691-2.332-2.426-2.332M144.658 26.715c0 1.074-.707 1.549-1.904 1.549-1.198 0-1.904-.475-1.904-1.549v-2.532c0-1.058.706-1.534 1.904-1.534 1.197 0 1.904.476 1.904 1.534v2.532Zm-1.904-5.065c-1.55 0-3.04.537-3.04 2.517v2.579c0 1.98 1.488 2.514 3.04 2.514s3.04-.536 3.04-2.514v-2.579c0-1.98-1.488-2.517-3.04-2.517ZM136.67 27.009l-1.688.935a1.728 1.728 0 0 1-.722.185c-.784 0-1.198-.445-1.198-1.457V24.14c0-1.027.707-1.456 1.796-1.456.707 0 1.812.32 1.812.32v4.005Zm0-5.034c-.445-.2-1.259-.324-1.796-.324-2.287 0-2.948 1.058-2.948 2.564v2.717c0 1.348.769 2.332 2.426 2.332.123 0 .293-.015.432-.062l1.889-1.32v1.212h1.135v-10.91h-1.135v3.791h-.003ZM130.046 21.826h-1.136V29.1h1.136v-7.274ZM130.046 18.186h-1.136v1.819h1.136v-1.819ZM91.753 18.186h-1.136v10.908h1.136V18.186ZM151.266 21.652c-.123 0-.29.015-.429.061l-1.889 1.32v-1.212h-1.136v7.274h1.136v-5.186l1.689-.934a1.68 1.68 0 0 1 .645-.18v-1.14h-.016M76.886 27.009l-1.688.934a1.728 1.728 0 0 1-.722.185c-.784 0-1.198-.444-1.198-1.456V24.14c0-1.028.707-1.457 1.797-1.457.706 0 1.811.322 1.811.322v4.004Zm0-5.034c-.444-.2-1.259-.324-1.796-.324-2.287 0-2.947 1.058-2.947 2.563v2.718c0 1.348.768 2.332 2.425 2.332.124 0 .293-.016.43-.062l1.888-1.32v1.212h1.136V18.186h-1.136v3.789ZM125.07 19.465l-1.135.794v6.78c0 1.657.706 2.21 2.071 2.21.267 0 .74-.052 1.007-.109v-1.05c-.241.03-.745.067-.899.067-.815 0-1.044-.46-1.044-1.29v-4.019h1.943v-1.027h-1.943v-2.358.002Z"
          fill="currentColor"
        />
      </svg>
    </div>
    <div
      class="c3"
    >
      <div
        aria-labelledby="contact"
        class="c4"
        data-testid="contact"
      >
        <div
          class="c5"
        >
          <p>
            Rua Desembargador Westphalen 15 conj. 602/603,
          </p>
          <p>
            Edif√≠cio Dante Aligheri, Centro CEP 80010-110
          </p>
          <p>
            CNPJ: 37.178.200/0001-05
          </p>
        </div>
        <nav
          class="c6"
        >
          <a
            href="https://www.instagram.com/hubdoinvestidor/"
            rel="noopenner, noreferrer"
            target="_blank"
          >
            <svg
              aria-hidden="true"
              class="c7"
              fill="none"
              focusable="false"
              stroke="currentColor"
              stroke-linecap="round"
              stroke-linejoin="round"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <rect
                height="20"
                rx="5"
                ry="5"
                width="20"
                x="2"
                y="2"
              />
              <path
                d="M16 11.37A4 4 0 1 1 12.63 8 4 4 0 0 1 16 11.37z"
              />
              <line
                x1="17.5"
                x2="17.51"
                y1="6.5"
                y2="6.5"
              />
            </svg>
          </a>
          <a
            href="https://twitter.com/hubinvestidor"
            rel="noopenner, noreferrer"
            target="_blank"
          >
            <svg
              aria-hidden="true"
              class="c7"
              fill="#F4F4F4"
              focusable="false"
              stroke="currentColor"
              stroke-linecap="round"
              stroke-linejoin="round"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M23 3a10.9 10.9 0 0 1-3.14 1.53 4.48 4.48 0 0 0-7.86 3v1A10.66 10.66 0 0 1 3 4s-4 9 5 13a11.64 11.64 0 0 1-7 2c9 5 20 0 20-11.5a4.5 4.5 0 0 0-.08-.83A7.72 7.72 0 0 0 23 3z"
              />
            </svg>
          </a>
          <a
            href="https://www.linkedin.com/company/hub-do-investidor/"
            rel="noopenner, noreferrer"
            target="_blank"
          >
            <svg
              aria-hidden="true"
              class="c7"
              fill="#F4F4F4"
              focusable="false"
              stroke="currentColor"
              stroke-linecap="round"
              stroke-linejoin="round"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M16 8a6 6 0 0 1 6 6v7h-4v-7a2 2 0 0 0-2-2 2 2 0 0 0-2 2v7h-4v-7a6 6 0 0 1 6-6z"
              />
              <rect
                height="12"
                width="4"
                x="2"
                y="9"
              />
              <circle
                cx="4"
                cy="4"
                r="2"
              />
            </svg>
          </a>
          <a
            href="https://www.facebook.com/hubdoinvestidoroficial"
            rel="noopenner, noreferrer"
            target="_blank"
          >
            <svg
              aria-hidden="true"
              class="c7"
              fill="#F4F4F4"
              focusable="false"
              stroke="currentColor"
              stroke-linecap="round"
              stroke-linejoin="round"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M18 2h-3a5 5 0 0 0-5 5v3H7v4h3v8h4v-8h3l1-4h-4V7a1 1 0 0 1 1-1h3z"
              />
            </svg>
          </a>
          <a
            href="https://www.youtube.com/channel/UC_aPpbiVd5eA0VgmBofw2Hg"
            rel="noopenner, noreferrer"
            target="_blank"
          >
            <svg
              aria-hidden="true"
              class="c7"
              fill="none"
              focusable="false"
              stroke="currentColor"
              stroke-linecap="round"
              stroke-linejoin="round"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M22.54 6.42a2.78 2.78 0 0 0-1.94-2C18.88 4 12 4 12 4s-6.88 0-8.6.46a2.78 2.78 0 0 0-1.94 2A29 29 0 0 0 1 11.75a29 29 0 0 0 .46 5.33A2.78 2.78 0 0 0 3.4 19c1.72.46 8.6.46 8.6.46s6.88 0 8.6-.46a2.78 2.78 0 0 0 1.94-2 29 29 0 0 0 .46-5.25 29 29 0 0 0-.46-5.33z"
              />
              <polygon
                points="9.75 15.02 15.5 11.75 9.75 8.48 9.75 15.02"
              />
            </svg>
          </a>
        </nav>
        <a
          href="https://api.whatsapp.com/send?phone=5516996148959"
          rel="noreferrer"
          target="_blank"
        >
          <button
            class="c8"
          >
            <span>
              Fale conosco
            </span>
          </button>
        </a>
      </div>
      <div
        aria-labelledby="resources"
        class="c4"
        data-testid="resources"
      >
        <nav
          class="c9"
          id="resources"
        >
          <a
            href="https://hubdoinvestidor.com.br/artigos"
          >
            Artigos
          </a>
          <a
            href="https://hubdoinvestidor.com.br/insights"
          >
            Hub Insights
          </a>
          <a
            href="https://hubdoinvestidor.com.br/#produtos"
          >
            Produtos
          </a>
          <a
            href="https://hubdoinvestidor.com.br/#quem-somos"
          >
            Quem somos
          </a>
          <a
            href="https://hubdoinvestidor.com.br/#fale-conosco"
          >
            Fale conosco
          </a>
          <a
            href="https://hubdoinvestidor.com.br/mb-termos-de-uso/"
          >
            Pol√≠tica de Privacidade e Termos de Uso
          </a>
        </nav>
        <img
          alt="Selos CVM e Apimec"
          class="c10"
          src="https://i.imgur.com/oDdiEzz.png"
        />
      </div>
    </div>
  </div>
  <div
    class="c11"
  >
    <div
      class="c1"
    >
      Copyright ¬© 
      2022
       Hub do Investidor | Todos os direitos reservados.
    </div>
  </div>
</footer>
`;
